<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="PARAMMEHTA-PM" name="Test" tests="1" failures="1" timestamp="2 May 2019 21:17:04 GMT" time="56.753" errors="0">
  <testcase name="getSortedPosts" time="56.698" classname="testFiles.RedittTest">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 5, Size: 5">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 5, Size: 5
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at testFiles.RedittTest.getSortedPosts(RedittTest.java:117)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- getSortedPosts -->
</testsuite> <!-- Test -->
